# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/l10n/tests/mochitest/dom_localization/test_setAttributes.html
# Commit: 595529cd906f
# Full Hash: 595529cd906f94cdb2bc12a7a40072db0c07198e
# Author: Zibi Braniecki <zbraniecki@mozilla.com>
# Date: 2021-08-03 22:01:19
# Regressor Bug: 1613705
# File Overlap Count: 1
# Description:
#   Bug 1613705 - [localization] part9: Update DOMLocalization to match Localization constructor. r=nika,emilio
#   
#   Depends on D113691
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D113692
# ==============================================================================

diff -r 9f3e2963d925 -r 595529cd906f dom/l10n/tests/mochitest/dom_localization/test_setAttributes.html
--- a/dom/l10n/tests/mochitest/dom_localization/test_setAttributes.html	Tue Aug 03 05:52:03 2021 +0000
+++ b/dom/l10n/tests/mochitest/dom_localization/test_setAttributes.html	Tue Aug 03 05:52:03 2021 +0000
@@ -7,8 +7,7 @@
   <link rel="stylesheet" type="text/css" href="chrome://mochikit/content/tests/SimpleTest/test.css">
   <script type="application/javascript">
   "use strict";
-
-  async function* generateBundles(resourceIds) {}
+  const l10nReg = new L10nRegistry();
 
   window.onload = function() {
     SimpleTest.waitForExplicitFinish();
@@ -16,7 +15,7 @@
     const domLoc = new DOMLocalization(
       [],
       false,
-      { generateBundles },
+      l10nReg,
     );
 
     const p1 = document.querySelectorAll("p")[0];